### PR TITLE
Add Javascript linter

### PR DESCRIPTION
**Pull request by TobyRet**
## Context

Now that we have specs for javascript on Apply, we need to add a linter to the pipeline to fix all the offences I have most likely caused in the process!

## Changes proposed in this pull request

- Add StandardJS linter
- Fix linter errors
- Run linter in the pipeline

## Link to Trello card

https://trello.com/c/UdWNqQTs/3208-lint-all-the-javascript-things

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] This code does not rely on the addition/removal of Azure config environment variables in the same Pull Request
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/docs/environment-variables.md#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by TobyRet** [comment]
@frankieroberto  @paulrobertlloyd  let me know if any changes need to be made to the JS linting rules 🔥

File: package.json [hunk 1]
```diff
@@ -14,14 +14,22 @@
...
   "devDependencies": {
     "jest": "^26.6.3",
-    "prettier": "^2.2.1"
+    "prettier": "^2.2.1",
+    "standard": "^16.0.3"
   },
   "scripts": {
-    "test": "jest"
+    "lint": "standard 'app/frontend/packs'",
+    "test": "npm run lint && jest"
```
Comment (vigneshmsft) [inline package.json:22]: `yarn lint && yarn jest` ?

**Review by paulrobertlloyd — APPROVED** [review]
This makes me sooooo happy 😁

File: Dockerfile [hunk 2]
```diff
@@ -57,7 +57,8 @@ FROM ${BASE_RUBY_IMAGE_WITH_GEMS_AND_NODE_MODULES} AS assets-precompile
 WORKDIR /app
 COPY . .
 
-RUN yarn jest && \
+RUN yarn run lint && \
+    yarn run test && \
```
Comment (duncanjbrown) [inline Dockerfile:61]: Should we add a script to `package.json` so this Dockerfile stays as dumb as possible?

**Review by duncanjbrown — APPROVED** [review]
Delicious autocorrect 🤤

File: Dockerfile (hunk 2 above)
Comment (duncanjbrown) [reply Dockerfile:61]: Oh, what @vigneshmsft said!

**Review by frankieroberto — APPROVED** [review]
👍  from me for standardjs.

**Pull request merged**